tools/do_release.sh

```diff
@@ -42,9 +42,27 @@ if [ "$GODOC_RELEASE_VERSION" == "" ]; then
 fi
 
 function updateVersionGo () {
-  echo package servenv > $ROOT/go/vt/servenv/version.go
-  echo  >> $ROOT/go/vt/servenv/version.go
-  echo "const versionName = \"$1)\"" >> $ROOT/go/vt/servenv/version.go
+  echo "/*
```

Comment:
Nit, but IMO a heredoc/herestring is nicer:
```
  cat <<EOF>${ROOT}/go/vt/servenv/version.go
/*
Copyright 2022 The Vitess Authors.

Licensed under the Apache License, Version 2.0 (the \"License\");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an \"AS IS\" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

package servenv

const versionName = "${1}"
EOF
```

As an aside, if you do more shell coding `shellcheck` is super handy (if you're not already aware of it).

Reply:
That's a great tip thank you. I had never heard of herestring before, only saw them!
I also downloaded and sanitized the script a little by using `shellcheck` 👍🏻